pom.xml

```diff
@@ -138,7 +138,7 @@
     <properties>
         <siddhi.version>5.1.5</siddhi.version>
         <siddhi.version.range>[5.0.0,6.0.0)</siddhi.version.range>
-        <wso2.transport.file.version>6.0.65</wso2.transport.file.version>
+        <wso2.transport.file.version>6.0.66-SNAPSHOT</wso2.transport.file.version>
```

Comment:
@Ajanthy why have we used a snapshot version over here?

Reply:
fixed in https://github.com/siddhi-io/siddhi-io-file/pull/95/commits/64b8c76e143422087753f3b4d9149b9ffb3024cd